config_manager/include/motor_initializer.h

```diff
@@ -0,0 +1,6 @@
+#ifndef MOTORLAND_FORK_MOTOR_INITIALIZER_H
+#define MOTORLAND_FORK_MOTOR_INITIALIZER_H
+
+Motor *init_motors(Config *config);
```

Comment:
Use prefix for function names and structures to avoid naming collisions and separate functions. For example, we can use module name as a prefix for all functions in the following way: `Motor *config_manager_init_motors()`